sql/hive/src/test/scala/org/apache/spark/sql/hive/CachedTableSuite.scala

```diff
@@ -34,6 +35,22 @@ import org.apache.spark.util.Utils
 class CachedTableSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
   import hiveContext._
 
+  private val originalCreateHiveTable = TestHive.conf.createHiveTableByDefaultEnabled
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    TestHive.conf.setConf(SQLConf.LEGACY_CREATE_HIVE_TABLE_BY_DEFAULT_ENABLED, true)
```

Comment:
do you know how cache can be broken if we don't create hive table?

Reply:
This suite use table `src` from TestHive. If we don't create hive table, it failed by:

`LOAD DATA is not supported for datasource tables: default.src;`

And many other tests across test suites failed due to the same reason.